config/locales/en/casework.yml

```diff
@@ -295,7 +312,31 @@ en:
     pays_council_tax:
       'yes': 'Yes'
       'no': 'No'
-
+    residential: Residential Property
+    commercial: Commercial Property
+    land: Land
+    house_type:
+      bungalow: Bungalow
+      detached: Detached
+      flat_or_maisonette: Flat or maisonette
+      semidetached: Semi-detached
+      terraced: Terraced
+      other: The type of property is not listed
+    property_type:
+      residential: Residential property
+      commercial: Commercial property
+      land: Land
+    relationship_type:
```

Comment:
The translation for other is missing
<img width="357" alt="Screenshot 2024-03-21 at 13 21 31" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/5ba00a34-0bfa-4901-8ef8-be2c40b83b6b">

Reply:
Need to confirm with designers